docs/user-guide/SettingsFile.md

```diff
@@ -27,6 +27,11 @@ Checkers' Friendly Names:
 * PayloadBody
 * Examples
 
+### client_certificate_path: str (default None)
+Path to your X.509 certificate file in PEM format.
+
+If provided and valid, RESTler will attempt to use it during the SSL handshake.
```

Comment:
Is this reference to "RESTler" okay, or should it be changed?

Reply:
Yes, this looks good.